src/commonComponents/primaryButton/PrimaryButton.js

```diff
@@ -0,0 +1,33 @@
+// prop-types
+import Proptypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+// css
+import "./primaryButton.css";
+
+const PrimaryButton = (props) => {
```

Comment:
you need to have a component called Button which can be primary, secondary, tertiary based on props